src/utils.ts

```diff
@@ -12,21 +12,6 @@ const findFilesWithEnding = (directory: string, endsWith: string): Array<string>
   return fs.readdirSync(directory).filter(file => file.endsWith(endsWith));
 };
 
-/**
- * Checks, if a string ends on one of the given endings
- * @param str - String to check the ending of
- * @param fileEndings - Set with all acceptable string endings
- * @returns - A boolean, that becomes true, if the string ends on the fileEnding Set
- */
-const resourceEndsWith = (str: string, fileEndings: Set<string>): boolean => {
-  const splitArray = str.split(".");
-  if (!splitArray.length || splitArray.length === 1) {
-    return false;
-  }
-  const urlEnd = splitArray[splitArray.length - 1];
-  return fileEndings.has(urlEnd);
-};
-
```

Comment:
Do we not need this anymore?

Reply:
I think we only needed it in the meme command. Also I haven't found any usage anywhere else. So I think it's good to go. :)